packages/react/fast-components-react-base/src/stack-panel/README.md

```diff
@@ -0,0 +1,39 @@
+## Stack Panel
+
+NOTE: this component is in initial development and should not yet be considered stable.
+
+The *stack panel* component is a container that enables virtualization of child elements arranged in a horizontal or vertical linear 'stack'. 
+
+### Usage
+
+Virtualization:  When virtualization is enabled the component does not render child items that are not in the visible area of the stackpanel's scrolling region or within the defined buffer zone (ie one or more items outside the viewport).
+
+Note that virtualized elements can't get focus so typically authors will want at least one out of view item instanciated otherwise it will not be possible to tab outside of the viewport.
+
+Authors should be aware that this component writes to the width/left or top/height css style attributes of children on render depending on the orientation of the stack panel.
+
+Keys: authors should strongly consider providing keys (https://reactjs.org/docs/lists-and-keys.html) to child elements to maintain stable identies for child items. 
+
+### Props
+
+ enableVirtualization:  Whether child elements are virtualized or not. Defaults to true.
```

Comment:
seems like the names of these props are now out of sync with the actual interface, can you update?